modules/velero-obtaining-by-downloading.adoc

```diff
@@ -23,16 +23,15 @@ The page includes instructions for:
 
 . Open a browser and navigate to link:https://{velero-domain}/docs/v{velero-version}/basic-install/#install-the-cli["Install the CLI" on the Verleo website].
 . Follow the appropriate procedure for macOS, GitHub, or Windows.
-. Download the Velero version appropriate for your version of OADP, according to the table that follows:
+. Download the Velero version appropriate for your version of OADP and {OCP} (OCP), according to the table that follows:
 +
-.OADP-Velero version relationship
-[cols="2", options="header"]
+.OADP-Velero-OCP version relationship
```

Comment:
Same issue here and below with the OCP/OKD. I wonder if `.OADP-Velero-{product-title} version` will work here.